react/week4/my-app/src/users.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import  {UserContext} from "./user.context";
+
+ const Users = () => {
+  
+    const {users, loading, error} = React.useContext(UserContext);
+  
+    if (error) {
+      return <div>{`error fetching ${error}...`}</div>;
+    }
+  
+    if (loading) {
+      return <div>Loading...</div>;
+    }
+  
+    if (!users || users.length === 0) {
+      return <div>No result..</div>;
+    }
+  
+    return <ul>
+        {users.map(user => {
```

Comment:
when a anonymous function has a single return, the code can be even more simplified:

```user => <li key={user.id}>{user.login}</li>``` 

(no curly brackets and no return)

Reply:
thanks for mentioning this 👍🏻